patches/api/0466-Expose-power-for-fireball-projectiles.patch

```diff
@@ -0,0 +1,41 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TonytheMacaroni <tonythemacaroni123@gmail.com>
+Date: Tue, 5 Mar 2024 20:43:00 -0500
+Subject: [PATCH] Expose power for fireball projectiles
+
+
+diff --git a/src/main/java/org/bukkit/entity/Fireball.java b/src/main/java/org/bukkit/entity/Fireball.java
+index 7a44707f2307dc4dbfea4de3f4baf3cc0490dc93..c638d81b9ca12c52dace2c53593ad2e5bc9fc116 100644
+--- a/src/main/java/org/bukkit/entity/Fireball.java
++++ b/src/main/java/org/bukkit/entity/Fireball.java
+@@ -23,4 +23,30 @@ public interface Fireball extends Projectile, Explosive {
+     @NotNull
+     public Vector getDirection();
+ 
++    /**
++     * Sets this entity's velocity in meters per tick
++     * <p>
++     * Note: For fireball entities, their movement is also controlled by their power.
++     *
++     * @see #setPower(Vector)
++     * @param velocity New velocity to travel with
++     */
++    @Override
++    public void setVelocity(@NotNull Vector velocity);
++
++    /**
++     * Sets the power of a fireball. The power determines the direction and magnitude of its acceleration.
++     *
++     * @param power the power
++     */
++    public void setPower(@NotNull Vector power);
++
++    /**
++     * Gets the power of a fireball. The power determines the direction and magnitude of its acceleration.
++     *
++     * @return the power
++     */
++    @NotNull
++    public Vector getPower();
++
```

Comment:
Missing `// Paper start` and `// Paper end` comments - https://github.com/PaperMC/Paper/blob/master/CONTRIBUTING.md